Library/Tracking/KSRAnalytics.swift

```diff
@@ -379,6 +379,26 @@ public final class KSRAnalytics {
     case watch
     case watched
 
+    /**
+     Initialize `TypeContext` with `DiscoveryParams`
```

Comment:
Rename to:
Initialize a `TypeContext` value with `DiscoveryParams` for use with discovery filters.